CMakeLists.txt

```diff
@@ -234,6 +234,7 @@ set( SUPERELASTIX_INSTALL_USE_FILE ${CMAKE_INSTALL_DIR}/UseSuperElastix.cmake )
 configure_file( SuperElastixConfig.cmake.in install/SuperElastixConfig.cmake @ONLY)
 configure_file( SuperElastixConfigVersion.cmake.in install/SuperElastixConfigVersion.cmake @ONLY)
 configure_file( UseSuperElastix.cmake.in install/UseSuperElastix.cmake @ONLY ) 
+configure_file(CMake/CTestCustom.cmake.in CTestCustom.cmake @ONLY)
```

Comment:
Should this not go in the build tree? I don't have much experience with CTest so I could very well be wrong.

Reply:
https://cmake.org/Wiki/CMake_Testing_With_CTest#Customizing_CTest says it should be in the build tree. Not sure if we customize our ctest already (is it different than default? @N-Dekker), but I can imagine that might need it at one point.

Reply:
I added CTestCustom.cmake.in to increase the maximum number of warnings, displayed at https://my.cdash.org/index.php?project=SuperElastix from 50 to 199 (CTEST_CUSTOM_MAXIMUM_NUMBER_OF_WARNINGS).